default/python/universe_generation/options.py

```diff
@@ -33,29 +33,30 @@
 # The following two options are used to determine the minimum number of systems and planets. This limit is
 # HS_MIN_SYSTEMS_IN_VICINITY systems and HS_MIN_PLANETS_IN_VICINITY_PER_SYSTEM planets per system within the near
 # vicinity of a home system, capped at HS_MIN_PLANETS_IN_VICINITY_TOTAL
-HS_MIN_SYSTEMS_IN_VICINITY = 8
-HS_MIN_PLANETS_IN_VICINITY_TOTAL = 10
-HS_MIN_PLANETS_IN_VICINITY_PER_SYSTEM = 1
+# HS_MIN_SYSTEMS_IN_VICINITY = 12
+# HS_MIN_PLANETS_IN_VICINITY_TOTAL = 12
+# HS_MIN_PLANETS_IN_VICINITY_PER_SYSTEM = 1
```

Comment:
If those constants are obsolete with the updated algorithm, just remove them instead of commenting them out. This just leads to commented out stuff accumulating in the code base. These values are already retained by the revision system.